src/fidocadj/dialogs/settings/SettingsManager.java

```diff
@@ -134,12 +136,15 @@ public int getInt(String key, int defaultValue)
      */
     public double getDouble(String key, double defaultValue)
     {
-        if (settings.containsKey(key)) {
-            return (double) settings.get(key);
+        String value = get(key, String.valueOf(defaultValue));
+        try {
+            return Double.parseDouble(value);
+        } catch (NumberFormatException e) {
+            // Se la conversione fallisce, restituisce il valore di default
```

Comment:
Do not leave the comment in Italian.

Reply:
I'll fix them right away, just some temporary comments that slipped through...